test.js

```diff
@@ -211,65 +211,128 @@ suite('scheduled rewards', () => {
       '0x000000000000000000000000000000000000dE12': '456621004566210048000000000000'
     })
   })
-  test('paid rewards', async t => {
-    {
-      const participants = ['0x000000000000000000000000000000000000dEa2']
-      const rewards = ['9132']
-      const res = await fetch(`${api}/paid`, {
-        method: 'POST',
-        body: JSON.stringify({
-          participants,
-          rewards,
-          signature: await sign(participants, rewards)
+  suite('paid rewards', () => {
```

Comment:
How about describing what endpoint this suite of tests is for?

```
suite('POST /paid', () => {
```

Reply:
```suggestion
  suite('POST /paid', () => {
```